content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,120 @@
+
+# Project Velocity
+
+The [CNCF K8s DevStats project](https://k8s.devstats.cncf.io/) aggregates a number of interesting data points related to the velocity of Kubernetes and various sub-projects. This includes everything from individual contributions to the number of companies that are contributing, and is a neat illustration of the depth and breadth of effort that goes into evolving this ecosystem.
+
+In the v1.21 release cycle, which ran for 12 weeks (January 11 to April 8), we saw contributions from [999 companies](https://k8s.devstats.cncf.io/d/9/companies-table?orgId=1&var-period_name=v1.20.0%20-%20now&var-metric=contributions) and [1279 individuals](https://k8s.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=v1.20.0%20-%20now&var-metric=contributions&var-repogroup_name=Kubernetes&var-country_name=All&var-companies=All).
+
+
+# Ecosystem Updates
+
+Master -> Main default branch migration [(PR#21451)](https://github.com/kubernetes/test-infra/pull/21451) is here! This migration divides tests as main and release branches, removes v1alpha2 tests and adds apidiff test, and fixes naming from master to main.
+
+
+# Event Updates
+
+- KubeCon + CloudNativeCon Europe 2021 will take place May 4 - 7, 2021! Registration opened on January 11. You can find more information about the conference [here](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/).
+- Kubernetes Community Days are being relaunched! Q1 will start with Africa and Bengaluru. You can find more information [here](https://kubernetescommunitydays.org/).
+
+# Upcoming release webinar
+
+Stay tuned for the upcoming release webinar happening this May.
+
+# Get Involved
+
+If you’re interested in contributing to the Kubernetes community, Special Interest Groups (SIGs) are a great starting point. Many of them may align with your interests! If there are things you’d like to share with the community, you can join the weekly community meeting, or use any of the following channels:
+
+* Find out more about contributing to Kubernetes at the new [Kubernetes Contributor website](https://www.kubernetes.dev/)
+* Follow us on Twitter [@Kubernetesio](https://twitter.com/kubernetesio) for latest updates
+* Join the community discussion on [Discuss](https://discuss.kubernetes.io/)
+* Join the community on [Slack](http://slack.k8s.io/)
+* Share your Kubernetes [story](https://docs.google.com/a/linuxfoundation.org/forms/d/e/1FAIpQLScuI7Ye3VQHQTwBASrgkjQDSS5TP0g3AXfFhwSM9YpHgxRKFA/viewform)
```

Comment:
I don't believe this is in use anymore. @oicheryl @kgamanji can you confirm?

Reply:
@mrbobbytables : This has been done. Would you please be able to take a look whenever free and let me know for any changes that need to be made? Thank you!